examples/demo.css

```diff
@@ -137,8 +137,9 @@ h1 {
 }
 #dwnld {
   max-width: 200px;
-  margin: 20px auto;
   margin-left: 5px;
```

Comment:
@JonathanXu1 Only removing this would push it in the center...have you tried removing `margin-left: 5px`? I guess there is no other change required...

Reply:
Yes there's the proper margins on line 140-142. https://github.com/publiclab/image-sequencer/blob/917831ed72f86593ac74ca7bf1ed9afaa8c1c92f/examples/demo.css#L138-L143